xdocs/index.xml

```diff
@@ -70,73 +70,53 @@ list of changes.
       </ul>
 
     <p>
-      In debian based Linux those dependencies could be installed by something like:
+        For Debian-based Linux distributions, these dependencies can be installed using the following command:
     </p>
-    <source>apt-get install libapr1.0-dev libssl-dev</source>
+    <source>apt-get install libapr1-dev libssl-dev openjdk-11-jdk</source>
     <p>
-      In rpm based Linux those dependencies could be installed by something like:
+        For RPM-based Linux distributions, these dependencies can be installed using the following command:
     </p>
-    <source>yum install apr-devel openssl-devel</source>
+    <source>yum install apr-devel openssl-devel java-11-openjdk-devel</source>
 </subsection>
 
 <subsection name="UNIX">
   <p>
-    On all the POSIX systems (Linux, Solaris, HP-UX, AIX etc...) a well-known
-    configure and make is used to build tc-native.<br/>
-    In the jni/native runs:
+      On all POSIX systems (Linux, Solaris, HP-UX, AIX etc...), the well-known
```

Comment:
Since we are here, turn it to "POSIX-like" only because GNU/Linux is not POSIX at all. That requires a certification.